src/nav_msg_glo.c

```diff
@@ -0,0 +1,285 @@
+/*
```

Comment:
Overall comment: please check that the code is formatted according to SN C coding style guide:
https://www.kernel.org/doc/Documentation/CodingStyle